setup.sh

```diff
@@ -121,6 +124,16 @@ fail () {
   exit 1
 }
 
+check_apis_enabled () {
+  SERVICES=`gcloud services list --enabled`
+  for API in 'Cloud Build API' 'Cloud Run Admin API' 'Cloud Pub/Sub API' 'Secret Manager API';
+    do
+      if [[ ${SERVICES} != *${API}* ]]; then
```

Comment:
Fix: we want this script to be as portable as possible, avoid `[[` which is bash-specific. The "To Search in Array" section here: https://www.geeksforgeeks.org/array-basics-shell-scripting-set-1/ might be portable. Otherwise, you could do the "matches via truncate to zero" trick.